File: apps/ios/PassAlarm/Domain/Entity/Occurrence.swift
import Foundation

struct Occurrence: Identifiable, Equatable, Sendable {
    var id: String { "\(planId.uuidString)-\(date)" }
    var planId: UUID
    var planLabel: String
    var date: String // "YYYY-MM-DD"
    var timeHHmm: String
    var soundId: String
    var fireDate: Date
    var isSkipped: Bool
    var skipReason: String?
}

File: apps/android/app/src/main/kotlin/com/nogeass/passalarm/domain/model/Occurrence.kt
package com.nogeass.passalarm.domain.model

data class Occurrence(
    val planId: Long,
    val planLabel: String,
    val date: String,
    val timeHHmm: String,
    val fireAtEpoch: Long,
    val isSkipped: Boolean = false,
    val skipReason: String? = null
)
